flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/stats/FlinkStatistic.scala

```diff
@@ -49,7 +49,7 @@ class FlinkStatistic(tableStats: Option[TableStats]) extends Statistic {
     * @return The stats of the specified column.
     */
   def getColumnStats(columnName: String): ColumnStats = tableStats match {
-    case Some(tStats) => tStats.colStats.get(columnName)
+    case Some(tStats) => tStats.getColStats.get(columnName)
```

Comment:
If we change the method names anyway, can we name them properly without abbreviations? `getColumnStats()` or `getMaxValue()` instead of just `getMax()`?